src/pipelines/local.py

```
@@ -976,6 +976,111 @@ async def generate(
+    ) -> AsyncIterator[str]:
+        """Stream LLM tokens from the local AI server.
+
+        Sends an ``llm_request`` with ``stream: true`` and yields tokens as
+        they arrive via ``llm_token`` messages.  Falls back to the non-streaming
+        ``generate()`` path if the server doesn't support streaming.
+        """
+        runtime_options = options or {}
+
+        try:
+            session = await self._ensure_session(call_id, runtime_options)
+        except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)